packages/@uppy/companion-client/src/RequestClient.js

```diff
@@ -116,15 +126,11 @@ export default class RequestClient {
     Subsequent requests use the cached result of the preflight.
     However if there is an error retrieving the allowed headers, we will try again next time
   */
-  async preflight (path) {
+  async preflight(path) {
     const allowedHeadersCached = allowedHeadersCache.get(this.hostname)
     if (allowedHeadersCached != null) return allowedHeadersCached
 
-    const fallbackAllowedHeaders = [
-      'accept',
-      'content-type',
-      'uppy-auth-token',
-    ]
+    const fallbackAllowedHeaders = ['accept', 'content-type', 'uppy-auth-token']
```

Comment:
a lot of unrelated changes - makes it hard to review. we should get our eslint config and prettier fixed